### PR TITLE
fix #3700 DropZone document 

### LINKED ISSUE #3700: There is a file type detection error bug in DropZone documents
## Issue summary
There was a bug in the DropZone documentation.
The `validImageTypes.indexOf(file.type) > 0` method fails to detect `image/gif` files when they are uploaded.

## Expected behavior

You should use `validImageTypes.indexOf(file.type) > -1`, because it will not detect `image/gif` type files when they are uploaded.

## Actual behavior

Change from `validImageTypes.indexOf(file.type) > 0` to `validImageTypes.indexOf(file.type) > -1`.

## Steps to reproduce the problem

1. upload *.gif file

### PR DESCRIPTION
**Pull request by malanjp**
### WHY are these changes introduced?

Fixes #3700 

### WHAT is this pull request doing?

Change from validImageTypes.indexOf(file.type) > 0 to validImageTypes.indexOf(file.type) > -1.

### CODE REVIEW

File: UNRELEASED.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Bug fixes
 
+- Fixed file type detection error in `DropZone` documents([ #3700 ](https://github.com/Shopify/polaris-react/issues/3700))
```
Comment (alex-page) [inline UNRELEASED.md:14]:
```suggestion
- Updated examples for `DropZone` so they accept all image types ([ #3700 ](https://github.com/Shopify/polaris-react/issues/3700))
```

**Review by alex-page — APPROVED** [review]
This is great @malanjp, one suggestion.

**Comment by BPScott** [comment]
[array.includes](https://caniuse.com/array-includes) is pretty ubiquitous these days. What do you think to using that instead?

```js
validImageTypes.includes(file.type)
```

**Comment by malanjp** [comment]
How to pass CI/Visual regression test?
```
Chromatic CLI v5.5.0
chromatic.com/docs/cli

✖ Missing project token

Sign in to chromatic.com/start and create a new project,
or find your project token on the Manage screen in an existing project.
Set your project token as the CHROMATIC_PROJECT_TOKEN environment variable
or pass the --project-token command line option.

ℹ Read more at chromatic.com/docs/setup
Error: non-zero exit code
```

**Review by alex-page — APPROVED** [review]

**Comment by alex-page** [comment]
Thankyou @malanjp ✨

**Pull request merged**